Formula/winetricks.rb

```diff
@@ -1,23 +0,0 @@
-class Winetricks < Formula
```

Comment:
This seems to be fine without unrar, judging from https://ports.macports.org/port/winetricks/summary